contributing/new-bindings.rst

```diff
@@ -0,0 +1,89 @@
+Adding support for other programming languages
+==============================================
+
+Ambition
+--------
+
+We'd like Xapian to be usable from as many languages as possible. However,
+this means more than just having bindings code. To truly support a language,
+we need to have:
+
+* working bindings that can be built against a latest version of Xapian and the
+  latest version of the target language
```

Comment:
It seems a bit odd to say "a latest version of Xapian", though I guess there are multiple latest versions (one per branch).

Reply:
Yeah, I meant that if someone contributes against 1.4.x but it doesn't work with master, that's still useful. It's not clear wording though, so maybe it should be "working bindings that can be built against a recent release of Xapian or git master, as well as against the latest version of the target language".